hardware_accelerators/rdma-remote-direct-memory-access.adoc

```diff
@@ -43,3 +43,19 @@ include::modules/rdma-configuring-the-nvidia-network-operator.adoc[leveloffset=+
 
 include::modules/rdma-configuring-the-gpu-operator.adoc[leveloffset=+1]
 
+// TELCODOCS-2232
+// include::modules/PREFORMATTED IMAGE.adoc[leveloffset=+1]
+
```

Comment:
@wabouhamad We need to add the topic here for "Downloading the preformatted image".

Reply:
@StephenJamesSmith I can't see this topic in the PR. Do you want to add it to the PR? If not, please resolve this thread and remove the comment on line 46